carball/json_parser/game.py

```diff
@@ -572,7 +573,13 @@ def parse_all_data(self, all_data) -> None:
                 player.parse_actor_data(_player_data)
             else:
                 # player not in endgame stats, create new player
-                player = Player().create_from_actor_data(_player_data, self.teams)
+                try:
+                    player = Player().create_from_actor_data(_player_data, self.teams)
+                except KeyError:
+                    # KeyError: 'Engine.PlayerReplicationInfo:Team'
```

Comment:
can we actually do a check for the specific error.

So that if there is a different type of key error later on we can catch it?

Reply:
done.